integrations/pagerduty/clients/pagerduty.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Dict, List
+
+import requests
```

Comment:
use httpx instead of requests for async usage of the api
and dont forget to add it to the project dependencies using 
`poetry add httpx`